lib/flop_phoenix.ex

```diff
@@ -1073,6 +1073,19 @@ defmodule Flop.Phoenix do
       list using row data. Note that these attributes will override any
       conflicting `tbody_td_attrs` that are set at the table level.
       """
+
+    attr :thead_label_component, :any,
```

Comment:
One thing I dislike about how this works is that currently you have to look at usage in the source to know which parameters are passed to the component. Let me know if there's a way you prefer I document the params. I'm open to feedback.